crates/rpc/src/v04/method/estimate_message_fee.rs

```diff
@@ -20,6 +20,7 @@ impl From<crate::v05::method::estimate_message_fee::EstimateMessageFeeError>
             ContractErrorV05 { revert_error } => {
                 Self::Internal(anyhow::anyhow!("Transaction reverted: {}", revert_error))
             }
+            Custom(error) => Self::Internal(error),
```

Comment:
Hmm, is this Custom -> Internal conversion right here?

Reply:
It is not, thanks!